tests/test_transformations.py

```diff
@@ -174,6 +195,38 @@ def test_split_polygon_on_antimeridian_ccw_east():
     ]
 
 
+def test_split_polygon_on_antimeridian_ccw_close_point():
+    """Polygon has a point that is extremely close to the antimeridian"""
+    polygon = Polygon([
+        (179.999999, 70.),
+        (179., 60.), (-170., 60.),
```

Comment:
Is there a reason why you are not adding 0 after the decimal?

Reply:
I think they started as integers but then there was maybe some sort of test failure, or maybe just pytest diff rendering showing them as different when they were not so I had to change them to floats. Easiest way was just adding the dots. I don't quite remember, it was a while ago.